apps/api/.env.example

```
@@ -6,6 +6,7 @@ DATABASE_URL=postgresql://postgres:YOUR_SUPABASE_PASSWORD@db.YOUR_PROJECT_REF.su
 # Server Configuration
 PORT=4000
 CORS_ORIGIN=http://localhost:3000
+JWT_SECRET=replace-with-a-long-random-secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_SECRET key should go before the PORT key

(UnorderedKey)